Jenkinsfile.d/core/package

```diff
@@ -66,6 +66,7 @@ pipeline {
   options {
     disableConcurrentBuilds()
     buildDiscarder logRotator(numToKeepStr: '15') // Retain only last 15 builds to reduce space requirements
+    retry(conditions: [agent(), kubernetesAgent(handleNonKubernetes: true), nonresumable()], count: 2)
```

Comment:
```suggestion
    retry(conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()], count: 2)
```
(The whole point of `handleNonKubernetes: true` is that you do _not_ also specify `agent`.)

Anyway I am not sure if this syntax even works—I have not tried it—and you are working too hard. The tested syntax for Declarative is

```groovy
agent {
  kubernetes {
    // …as before
    retries 2
  }
}
```
applied above to https://github.com/jenkins-infra/release/blob/e2368e9b9ea154589eb8192d3922a7da9624e856/Jenkinsfile.d/core/package#L2-L7 (GH suggestions do not allow this)